src/main/java/org/stuartgunter/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -163,6 +168,9 @@
     @NotNull
     private ProtocolOptions.Compression compression = ProtocolOptions.Compression.NONE;
 
+    @NotNull
+    private Duration maxSchemaAgreementWait = Duration.seconds(0);
```

Comment:
Please can this default to null. This would make it consistent with other config properties, and more accurately represent the fact that it has not ben set. Magic numbers aren't good practice.